src/General/html/Settings/Advanced.html

```diff
@@ -110,6 +110,13 @@
 </fieldset>
 
 <fieldset>
+    <legend>Additional Cooldown Time</legend>
+    <div>
+        Seconds: <input type=number name=additionalCooldown class=field min=1 value=#{Conf['additionalCooldown']}>
```

Comment:
I want to avoid inserting unescaped Javascript expressions into the HTML, and it's only because of a bug (now fixed) that this worked.  And in this case the value is set in `src/General/Settings.coffee` anyway.